### PR TITLE
Add cosmos-sdk project

### PR DESCRIPTION
**Pull request by cuonglm**

### CODE REVIEW

**Comment by odeke-em** [comment]
Thank you @cuonglm!

For reviewers, for context, the cosmos-sdk is used for a currency of a [$4-5 Billion USD market cap as of April 18th 2021](https://coinmarketcap.com/currencies/cosmos/), it is heavily relied upon by an ecosystem securing [at least $114 Billion USD as of April 18th 2021](https://cosmos.network/ecosystem/tokens/) It is relied upon by a whole bunch of projects too. Thank you!

File: projects/cosmos-sdk/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Google Inc.
```
Comment (AdamKorcz) [inline projects/cosmos-sdk/Dockerfile:1]: `2021 Google LLC` instead of `2021 Google Inc.`

File: projects/cosmos-sdk/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+# Copyright 2021 Google Inc.
```
Comment (AdamKorcz) [inline projects/cosmos-sdk/build.sh:2]: `2021 Google LLC` instead of `2021 Google Inc.`

**Review by asraa — APPROVED** [review]
Thanks for linking the issues! I see a lot more fuzzers in the branch, excited to see them added soon.

**Comment by cuonglm** [comment]
@asraa We're going to [make a script](https://github.com/cosmos/cosmos-sdk/pull/9140) instead of adding more fuzzing by creating new PR in oss-fuzz repo, is it best practice?

> I see a lot more fuzzers in the branch, excited to see them added soon.

Yes, there're more fuzzers in `fuzz-packages` branch, but we need to make some adjustment to make it works with `compiler_go_fuzzer` script.

**Comment by asraa** [comment]
> @asraa We're going to make a script instead of adding more fuzzing by creating new PR in oss-fuzz repo, is it best practice?

If i understand correctly, then yes. Ideally, all the fuzzers live in cosmos main repo, and yes, you can reference the build script in your dir in `build.sh`

**Comment by cuonglm** [comment]
> > @asraa We're going to make a script instead of adding more fuzzing by creating new PR in oss-fuzz repo, is it best practice?
> 
> If i understand correctly, then yes. Ideally, all the fuzzers live in cosmos main repo, and yes, you can reference the build script in your dir in `build.sh`

Thanks FYI :+1: I added a PR for that https://github.com/google/oss-fuzz/pull/5638

**Comment by cuonglm** [comment]
Hi @asraa, sorry for hijack you again. So I changed all cosmos-sdk current fuzzer to work with compile_go_fuzzer script in https://github.com/cosmos/cosmos-sdk/commit/f55112814d9e3cd1fd3b9837e3e9dc8fe148925d

How can I know whether the change is good or bad (e.g, break oss-fuzz build or not)?

**Comment by asraa** [comment]
You can check builds here https://oss-fuzz-build-logs.storage.googleapis.com/index.html#cosmos-sdk that run daily

but since the build script lives in your repo you won't see the presubmit on a PR change (I think for that reason most projects have oss-fuzz specific build scripts living in oss-fuzz, but code living in their repo that may run in their CI as presubmit)

**Pull request merged**